plugins/plugin.h

```diff
@@ -380,6 +380,17 @@ struct janus_callbacks {
 	 * @param[in] event The event to notify as a Jansson json_t object */
 	void (* const notify_event)(janus_plugin *plugin, janus_plugin_session *handle, json_t *event);
 
+	/*! \brief Method to check whether a signed token is valid
+	 * \note accepts only tokens with the plugin identifier as realm
+	 * @param[in] token The token to validate
+	 * @returns TRUE if the signature is valid and not expired */
```

Comment:
Please add

    , and FALSE otherwise

I know it's implicit and redundant, but it's to adhere to the convention of the other doxygen-generated docs we have.